res/css/style.css

```diff
@@ -193,6 +193,43 @@ body, #root, .profileViewer {
   color: highlighttext;
 }
 
+.treeViewCategoryKnob {
+  display: inline-block;
```

Comment:
as an inline-block, I find it's slightly too high compared to the text and the expanding arrow. Maybe it needs `vertical-align: middle` ? I tried with this and it's not satisfying either... so maybe don't do anything :)

also some of them are too small on my computer, depending on the zoom level, see:
![screenshot](https://user-images.githubusercontent.com/454175/41107762-4595b7c8-6a73-11e8-8053-a6e32b5467bb.png)
The purple one misses some height at some zoom levels...
But let's not spend too much time on this if you don't have a good idea to fix it.

Reply:
Ensuring that something is square at a non-integer devicePixelRatio is basically impossible if you don't want to resort to an image. I'm not going to try to fix this.